charts/k8ssandra/templates/grafana/cassandra-overview.configmap.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.monitoring.grafana.provision_dashboards }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-k8ssandra-grafana-cassandra-overview-dashboard
```

Comment:
What do you think about shortening the name to `{{ .Release.Name }}-cassandra-overview`?